### PR TITLE
Update _index.en.md

### PR DESCRIPTION
**Pull request by lunamidori5**
Added a small change to the docker file under Cublas as a test, you can remove this if needed.
Just wanted to see how easy it is to update the site.

### CODE REVIEW

File: content/getting_started/_index.en.md [hunk 1]
```diff
@@ -160,6 +160,35 @@ Note: the binary inside the image is pre-compiled and might not suite all the CP
 
 Requirement: nvidia-container-toolkit (installation instructions [1](https://www.server-world.info/en/note?os=Ubuntu_22.04&p=nvidia&f=2) [2](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html))
 
+To get started using Cuda GPU Update the docker file.
```
Comment (mudler) [inline content/getting_started/_index.en.md]:
```suggestion
To get started using CUDA GPU with `docker-compose`, see the following example:
```

File: content/getting_started/_index.en.md [hunk 2]
```diff
@@ -160,6 +160,35 @@ Note: the binary inside the image is pre-compiled and might not suite all the CP
...
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+    image: quay.io/go-skynet/local-ai:master-cublas-cuda12
+    build:
+      context: .
+      dockerfile: Dockerfile
+    tty: true # enable colorized logs
+    restart: always # should this be on-failure ?
```
Comment (mudler) [inline content/getting_started/_index.en.md]: Better to not leave open questions in the example
```suggestion
    restart: always # or also on-failure
```

File: content/getting_started/_index.en.md [hunk 3]
```diff
@@ -160,6 +160,35 @@ Note: the binary inside the image is pre-compiled and might not suite all the CP
...
+    ports:
+      - 8080:8080
+    env_file:
+      - .env
```
Comment (mudler) [inline content/getting_started/_index.en.md]:
```suggestion
```

File: content/getting_started/_index.en.md [hunk 4]
```diff
@@ -160,6 +160,35 @@ Note: the binary inside the image is pre-compiled and might not suite all the CP
...
+              capabilities: [gpu]
+    image: quay.io/go-skynet/local-ai:master-cublas-cuda12
+    build:
+      context: .
+      dockerfile: Dockerfile
+    tty: true # enable colorized logs
+    restart: always # should this be on-failure ?
+    ports:
+      - 8080:8080
+    env_file:
```
Comment (mudler) [inline content/getting_started/_index.en.md]:
```suggestion
```

File: content/getting_started/_index.en.md [hunk 5]
```diff
@@ -160,6 +160,35 @@ Note: the binary inside the image is pre-compiled and might not suite all the CP
...
+  api:
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+    image: quay.io/go-skynet/local-ai:master-cublas-cuda12
+    build:
```
Comment (mudler) [inline content/getting_started/_index.en.md]:
```suggestion
```

File: content/getting_started/_index.en.md [hunk 6]
```diff
@@ -160,6 +160,35 @@ Note: the binary inside the image is pre-compiled and might not suite all the CP
...
+      context: .
```
Comment (mudler) [inline content/getting_started/_index.en.md]:
```suggestion
```

File: content/getting_started/_index.en.md [hunk 7]
```diff
@@ -160,6 +160,35 @@ Note: the binary inside the image is pre-compiled and might not suite all the CP
...
+      dockerfile: Dockerfile
```
Comment (mudler) [inline content/getting_started/_index.en.md]:
```suggestion
```

File: content/getting_started/_index.en.md [hunk 8]
```diff
@@ -183,7 +183,6 @@ services:
     ports:
       - 8080:8080
     env_file:
-      - .env
```
Comment (lunamidori5) [inline content/getting_started/_index.en.md]: this is needed to load the env file...

File: content/getting_started/_index.en.md [hunk 9]
```diff
@@ -182,7 +182,6 @@ services:
     restart: always # or also on-failure
     ports:
       - 8080:8080
-    env_file:
```
Comment (lunamidori5) [inline content/getting_started/_index.en.md]: This is needed to load the users env file...?

**Review by lunamidori5 — COMMENTED** [review]
They still need to load the Env file to pick their settings...

**Pull request closed without merging**